lib/util.js

```diff
@@ -34,15 +44,16 @@ exports.parseFilters = json => {
 
 exports.parseBody = body => {
   const str = between(body, 'var ytInitialData =', '; \n');
-  const apiKey = between(body, 'INNERTUBE_API_KEY":"', '"') || OLD_UTIL.between(body, 'innertubeApiKey":"', '"');
-  const clientVersion = between(body, 'INNERTUBE_CONTEXT_CLIENT_VERSION":"', '"') || OLD_UTIL.between(body, 'innertube_context_client_version":"', '"');
+  const apiKey = between(body, 'INNERTUBE_API_KEY":"', '"') || between(body, 'innertubeApiKey":"', '"');
+  const clientVersion = between(body, 'INNERTUBE_CONTEXT_CLIENT_VERSION":"', '"') || between(body, 'innertube_context_client_version":"', '"');
+  DEFAULT_CONTEXT.client.clientVersion = clientVersion;
   return { json: JSON.parse(str || null), apiKey, clientVersion };
 };
 
 // Parsing utility
 const parseText = exports.parseText = txt => txt.simpleText || txt.runs.map(a => a.text).join('');
 
-exports.parseNumFromText = txt => Number(parseText(txt).replace(/[^0-9.,]+/, ''));
```

Comment:
🤔 now i know why that returns NaN
dumb `.` and `,`
guess `\D` solves the Problem